etc/infrastructure/php/conf.d/xdebug.ini

```diff
@@ -4,7 +4,7 @@ zend_extension=xdebug.so
 xdebug.remote_enable = 1;
 xdebug.remote_connect_back = 1;
 xdebug.remote_autostart = 1;
-xdebug.remote_port = 9001;
+xdebug.remote_port = 9000;
```

Comment:
Doesn't conflict with the fpm?

Reply:
Oooop! My bad. Good one! 🙂

I've just fixed it in the latest commit 🤟